packages/staking/src/features/Drawer/StakePoolDetail.tsx

```diff
@@ -387,20 +403,23 @@ export const StakePoolDetailFooter = ({ popupView }: StakePoolDetailFooterProps)
 
   const actionButtons = useMemo(
     () =>
-      makeActionButtons(t, {
-        addStakingPool: ableToSelect && !selectionsEmpty && { callback: onSelectClick },
-        manageDelegation: poolInCurrentPortfolio && { callback: onManageDelegationClick },
-        selectForMultiStaking: ableToSelect && selectionsEmpty && { callback: onSelectClick },
-        stakeOnThisPool: selectionsEmpty && ableToStakeOnlyOnThisPool && { callback: onStakeOnThisPool },
-        unselectPool: poolSelected && { callback: onUnselectClick },
-      }),
+      !isSharedWallet
+        ? makeActionButtons(t, {
+            addStakingPool: ableToSelect && !selectionsEmpty && { callback: onSelectClick },
```

Comment:
WTF is this 😨 I know you just followed it, I am the original author of this code. It must have been a late late night when I created this monster...

However I think that the correct way here would be to use that original `makeActionButtons` function and adjust conditions disabling all unnecessary buttons, so here it would be
```suggestion
            addStakingPool: !isShraredWallet && ableToSelect && !selectionsEmpty && { callback: onSelectClick },
```

Reply:
done here https://github.com/input-output-hk/lace/pull/1311/files